config/plugin-commands-config/src/configSet.ts

```diff
@@ -34,10 +35,44 @@ export async function configSet (opts: ConfigCommandOptions, key: string, value:
   }
   key = camelCase(key)
   await updateWorkspaceManifest(opts.workspaceDir ?? opts.dir, {
-    [key]: value,
+    [key]: castField(value, kebabCase(key)),
   })
 }
 
+function castField (value: any, key: string) { // eslint-disable-line
+  if (typeof value !== 'string') {
+    return value
+  }
+
+  // @ts-expect-error
+  const typeList = [].concat(types[key])
+  // @ts-expect-error
+  const isNumber = typeList.includes(Number)
+
+  value = `${value}`.trim()
```

Comment:
As has been judged above, if value is not a string, it will not come here, so the template literals here should not be necessary.

```suggestion
  value = value.trim()
```